<?xml version="1.0"?>
<launch>

  <node name="planner" pkg="robotics_challenge" type="planner_node.py" output="screen">
    <rosparam command="load" file="$(find robotics_challenge)/cfg/problem3.yaml" />

    <!--
    <remap from="costmap_2d/costmap/costmap" to=""/>
    -->
  </node>

</launch>

